.gitmodules

```diff
@@ -9,6 +9,7 @@
 [submodule "eosio_llvm"]
 	path = eosio_llvm
 	url = https://github.com/eosio/llvm
+	branch = qy-read-only-action
```

Comment:
This branch should be the 'eosio' branch of llvm.  So, please make a PR to LLVM to the branch 'eosio' and then update this.

Reply:
https://github.com/EOSIO/llvm/pull/29 was just created, which updates clang to the head of its 'eosio' branch (with `read-only` attribute added). The `branch` setting here will be removed after the LLVM PR is merged.